compiler/runtime/palloc.c

```diff
@@ -208,8 +208,8 @@ int SKIP_cond_timedwait(pthread_cond_t* x, pthread_mutex_t* y, uint32_t secs) {
   return pthread_cond_timedwait(x, y, &ts);
 }
 
-void SKIP_cond_broadcast(void* c) {
-  pthread_cond_broadcast(c);
+int32_t SKIP_cond_broadcast(void* c) {
+  return pthread_cond_broadcast(c);
```

Comment:
```suggestion
  return (int32_t)pthread_cond_broadcast(c);
```
Just to make it explicit we're not keeping the same type.